TRAMS-API/Models/Downstream/D365/SearchProjectsD365PageModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Models.Downstream.D365
+{
+    public class SearchProjectsD365PageModel
+    {
+        public List<SearchProjectsD365Model> Projects { get; set; }
+
+        public int CurrentPage { get; set; }
```

Comment:
Suggestion:  Have the paging properties on a base class so that they can be used for future searching scenarios without repetition.

Reply:
Refactored